[Rewrite] Function Wikis - #2 (#203)
NilPointer-Software/bdfd-wiki: src/bdscript/url.md

# $url

> URL encodes or decodes a provided text.

Reviewer: ```suggestion
Encodes or decodes the provided text in the URL encoding format.
```

> URL encoding is a method of converting of reserved, unsafe, and non-ASCII characters in URLs to a format that is universally accepted and understood by all web browsers and servers while URL decoding is the vice-versa of URL encoding.

Reviewer: ```suggestion
URL encoding is a method of converting reserved, unsafe, or non-ASCII characters to a URL format that is universally accepted and understood by all web browsers and servers while URL decoding is the vice-versa of URL encoding.
```
